src/hdmf/region.py

```diff
@@ -0,0 +1,91 @@
+from abc import ABCMeta, abstractmethod
```

Comment:
Not sure if we already have specific tests for RegionSlicer and ListSlicer but if so, they should probably also move to a corresponding file for testing this module.

Reply:
I don't think we do